pwnlib/libcdb.py

```diff
@@ -88,6 +88,9 @@ def search_by_build_id(hex_encoded_id):
         Path to the downloaded library on disk, or :const:`None`.
 
     Examples:
+    .. doctest::
+       :skipif: offline
```

Comment:
Since we are skipping all of the tests in the entire module if offline, we should use a module-level `__test__`.  See the `doctests` docs for more information.